cmd/loadtest/main.go

```diff
@@ -25,9 +25,22 @@ import (
 // ClientTimeout is the Horizon HTTP request timeout.
 const ClientTimeout = 120 * time.Second
 
+// Support arrays of string flags
+type arrayFlags []string
+
+func (i *arrayFlags) String() string {
+	return "my string representation"
```

Comment:
you should print the array's contents here. otherwise if you print this object it will always output "my string representation"

Reply:
I copy+pasted that code as a hack for array support for flags.  I don't actually care about useful help messages here.